includes/Experiments.php

```diff
@@ -312,14 +312,26 @@ public function get_experiments(): array {
 				'default'     => true,
 			],
 			/**
-			 * Author: @brittanyirl
-			 * Issue: #10115
-			 * Creation date: 2022-02-02
+			 * Author: @miina
+			 * Issue: #9880
+			 * Creation date: 2021-12-15
+			 */
+			[
+				'name'        => 'customFonts',
+				'label'       => __( 'Custom Fonts', 'web-stories' ),
+				'description' => __( 'Enable adding custom fonts', 'web-stories' ),
+				'group'       => 'general',
+				'default'     => true,
+			],
+			/**
+			 * Author: @spacedmonkey
+			 * Issue: #8821
+			 * Creation date: 2022-01-19
 			 */
 			[
-				'name'        => 'enableUpdatedPublishStoryModal',
-				'label'       => __( 'Updated Publish Story Modal', 'web-stories' ),
-				'description' => __( 'Enable new pre-publish confirmation modal', 'web-stories' ),
+				'name'        => 'enhancedPageBackgroundAudio',
+				'label'       => __( 'Page Background Audio', 'web-stories' ),
+				'description' => __( 'Enable adding captions to background audio', 'web-stories' ),
 				'group'       => 'editor',
 				'default'     => true,
 			],
```

Comment:
No idea what happened here but I just opened #11315 to fix this

Reply:
what the heck!! thank you.